backend/src/main/java/com/google/minicrm/data/Form.java

```diff
@@ -17,91 +17,73 @@
 import com.google.appengine.api.datastore.Entity;
 import com.google.appengine.api.datastore.Key;
 import com.google.appengine.api.datastore.KeyFactory;
-import com.google.appengine.api.users.User;
-import com.google.minicrm.utils.AdvertiserUtil;
 import java.util.Date;
 
 /**
```

Comment:
can you also detail in the javadoc the relationship between a Form and a lead, and how the dataflow works?